tests/common/snappi_tests/common_helpers.py

```diff
@@ -471,22 +471,39 @@ def config_wred(host_ans, kmin, kmax, pmax, profile=None, asic_value=None):
     return True
 
 
-def enable_ecn(host_ans, prio, asic_value=None):
+def enable_ecn(host_ans, prio, asic_value=None, profile=None):
     """
-    Enable ECN marking on a priority
+    Enable ECN marking on a priority for specified profiles
 
     Args:
         host_ans: Ansible host instance of the device
         prio (int): priority
         asic_value: asic value of the host
+        profile (str): name of profile to configure (None means all profiles)
 
     Returns:
-        N/A
+        True if configuration succeeds, False otherwise
     """
-    if asic_value is None:
-        host_ans.shell('sudo ecnconfig -q {} on'.format(prio))
-    else:
-        host_ans.shell('sudo ip netns exec {} ecnconfig -q {} on'.format(asic_value, prio))
+    profiles = get_wred_profiles(host_ans, asic_value)
```

Comment:
Let's discuss more on the changes in this PR offline. We shouldn't see such error messages in the log analyzer unless the user manually tried to configure in the incorrect format

Reply:
Found the issue. https://github.com/sonic-net/sonic-utilities/pull/2889 needs to be backported to 202305